readme.md

```diff
@@ -1165,6 +1168,8 @@
 | [Colorpicker](https://colorpicker.fr)| Open Source colors software: Retrieve, manipulate and store your colors easily! |
 | [Google Web Designer](https://webdesigner.withgoogle.com/)| Create engaging, interactive HTML5-based designs and motion graphics that can run on any device. |
 | [Origami Studio](https://origami.design)| Interactive interface design tool created by Facebook: For mac only |
+| [LibreOffice - Draw](https://www.libreoffice.org/discover/draw/) | Draw lets you produce anything from a quick sketch to a complex plan, and gives you the means to communicate with graphics and diagrams. With a maximum page size of 300cm by 300cm, Draw is a an excellent package for producing technical drawings, general posters, and much else. |
+| [Wings 3D](http://www.wings3d.com/) | Wings 3D is a subdivision modeler inspired by Nendo and Mirai from Izware. It is possible to assign materials, vertex color, UV coordinates and textures, but there will be improvements in those features before Wings goes 1.0. |
```

Comment:
Please unlint the top portion of the file and if you can please one PR per resource :)